splinterd/src/daemon.rs

```diff
@@ -1586,7 +1610,13 @@ pub enum CreateError {
     MissingRequiredField(String),
 }
 
-impl Error for CreateError {}
+impl Error for CreateError {
```

Comment:
This change looks unnecessary

Reply:
Removed, it was left over after a previous change.